pandas/tests/dtypes/cast/test_promote.py

```diff
@@ -489,9 +450,9 @@ def test_maybe_promote_any_with_timedelta64(
     _check_promote(dtype, fill_value, expected_dtype, exp_val_for_scalar)
 
 
-def test_maybe_promote_string_with_any(string_dtype, any_numpy_dtype_reduced):
+def test_maybe_promote_string_with_any(string_dtype, any_numpy_dtype):
```

Comment:
is the idea here just that `any_numpy_dtype` is a superset of `any_numpy_dtype_reduced`, and that one might as well test over the larger set?

Reply:
Yup, exactly